src/Components/Form.js

```diff
@@ -24,11 +25,11 @@ const Form = () => {
 
   return (
     <section className="add-new-book-section">
-      <h2>ADD NEW BOOK</h2>
-      <form onSubmit={(e) => onSubmit(e)}>
-        <input onChange={handleChange('title')} type="text" name="title" placeholder="Book title" value={bookInfo.title} />
-        <input onChange={handleChange('category')} type="text" name="category" placeholder="Author name" value={bookInfo.category} />
-        <button type="submit">ADD BOOK</button>
+      <h2 className="add-book">ADD NEW BOOK</h2>
+      <form onSubmit={(e) => onSubmit(e)} className="form">
+        <input onChange={handleChange('title')} type="text" name="title" placeholder="Book title" value={bookInfo.title} className="title" />
+        <input onChange={handleChange('category')} type="text" name="category" placeholder="Author name" value={bookInfo.category} className="category" />
```

Comment:
- [ ] Kindly note that this field should be a selector input, not text input. 

![image](https://user-images.githubusercontent.com/65068771/155845483-50910c0f-4b16-472b-aa14-70a7da49c4b7.png)

Reply:
Yes, but the project builds based on input text from the beginning 😉